roles/redhat_packages/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: gather redhat-packages fact data
+  raw: rpm -qa --qf "%{NAME}|%{VERSION}|%{RELEASE}|%{INSTALLTIME}|%{VENDOR}|%{BUILDTIME}|%{BUILDHOST}|%{SOURCERPM}|%{LICENSE}|%{PACKAGER}|%{INSTALLTIME:date}|%{BUILDTIME:date}|%{DSAHEADER:pgpsig}|%{RSAHEADER:pgpsig}|%{SIGGPG:pgpsig}|%{SIGPGP:pgpsig}|\n"
+  register: redhat_packages_results
+  ignore_errors: yes
+  when: internal_have_rpm
+
+- name: add redhat-packages.results to dictionary
```

Comment:
clean up task name to reflect more of value extraction.

Reply:
👌